patches/chrome-browser-resources-settings-appearance_page-appearance_page.html.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/chrome/browser/resources/settings/appearance_page/appearance_page.html b/chrome/browser/resources/settings/appearance_page/appearance_page.html
+index 720d25d126a118315a08cef90d77d1fc78775254..18513bd02052b4a76bca85dd0951c14e4e3b6665 100644
+--- a/chrome/browser/resources/settings/appearance_page/appearance_page.html
++++ b/chrome/browser/resources/settings/appearance_page/appearance_page.html
+@@ -19,6 +19,8 @@
+ <link rel="import" href="appearance_fonts_page.html">
+ <link rel="import" href="home_url_input.html">
+ 
++<link rel="import" href="../brave_appearance_page/brave_appearance_page.html">
++
+ <dom-module id="settings-appearance-page">
+   <template>
+     <style include="settings-shared md-select iron-flex">
+@@ -175,6 +177,7 @@
+             </template>
+           </select>
+         </div>
++        <settings-brave-appearance-page></settings-brave-appearance-page>
```

Comment:
Please can we put this at the top of the appearance section? So that it is next to the extension theme item.

Reply:
done.